File: c programming/transpose.c
#include <stdio.h>
int main()
{
    int r, c, i, j, a[100][100], trans[100][100];
    printf("Enter row and columns:");
    scanf("%d %d", &r, &c);
    printf("Enter elements:");
    for (i = 0; i < r; i++)
        for (j = 0; j < c; j++)
        {
            scanf("%d", &a[i][j]);
        }
    for (i = 0; i < r; i++)
        for (j = 0; j < c; j++)
        {
            trans[j][i] = a[i][j];
        }
    printf("transpose of a matrix is:");
    for (i = 0; i < r; i++)
    {
        for (j = 0; j < c; j++)
        {
            printf("%d\t", trans[i][j]);
        }
        printf("\n");
    }
}

File: PASCAL/transpose.pas
var 
a:array[1..10,1..10]of integer;
i,j,r,c:integer;
begin
    writeln('eneter size of row:');
    readln(r);
    writeln('enter size of column:');
    readln(c);
    writeln('Enter elements:');
    for i:=1 to r do 
    for j:=1 to c do 
    readln(a[i][j]);
    writeln('the giver matrix is:');
    for i:=1 to r do 
    begin 
    for j:=1 to c do 
    write(#9,a[i][j]);
    writeln(#10);
    end;
    writeln('transpose of a matrix:');
    for j:=1 to r do
    begin 
    for i:=1 to c do 
    write(#9,a[i][j]);
    writeln(#10);
    end;
end.